frontend/app/src/App.test.tsx

```diff
@@ -219,1619 +369,1738 @@ describe("App", () => {
           userInfo: {},
           sessionStatus: {},
         },
-      }
-
-      app.handleMessage(fwMessage)
+      })
 
       expect(window.location.reload).toHaveBeenCalled()
     })
   })
 
-  it("starts screencast recording when the MainMenu is clicked", () => {
```

Comment:
Just confirming: These two tests were removed as they are redundant right? (Seems like covered in `withScreencast.test.tsx`)